decidim-assemblies/app/views/decidim/assemblies/admin/assembly_user_roles/index.html.erb

```diff
@@ -9,52 +9,76 @@
               <% end %>
             </td>
-            <td>
+            <td data-label="<%= t("models.user.fields.role", scope: "decidim.admin") %>">
               <%= t("#{role.role}", scope: "decidim.admin.models.assembly_user_role.roles") %><br>
             </td>
-            <td class="table-list__actions">
-              <% if allowed_to?(:invite, :assembly_user_role, user_role: role) && role.user.invited_to_sign_up? %>
-                <%= icon_link_to "refresh-line", resend_invitation_assembly_user_role_path(current_assembly, role), t("actions.resend_invitation", scope: "decidim.admin"), class: "resend-invitation", method: :post %>
-              <% end %>
+            <td class="table-list__actions" data-label="<%= t("models.assembly_user_role.fields.actions", scope: "decidim.admin") %>">
+              <button type="button" data-component="dropdown" data-target="actions-assembly-user-role-<%= role.id %>" aria-label="<%= t("decidim.admin.actions.actions_label", resource: role.user.name) %>">
+                <%= icon "more-fill", class: "text-secondary" %>
+              </button>
 
-              <% if allowed_to? :update, :assembly_user_role, user_role: role %>
-                <%= icon_link_to "pencil-line", edit_assembly_user_role_path(current_assembly, role), t("actions.edit", scope: "decidim.admin"), class: "action-icon--edit" %>
-              <% end %>
+              <div class="inline-block relative">
+                <ul id="actions-assembly-user-role-<%= role.id %>" class="dropdown dropdown__action" aria-hidden="true">
+                  <% if allowed_to?(:invite, :assembly_user_role, user_role: role) && role.user.invited_to_sign_up? %>
+                    <li class="dropdown__item">
+                      <%= link_to resend_invitation_assembly_user_role_path(current_assembly, role), method: :post, class: "dropdown__button resend-invitation" do %>
+                        <%= icon "refresh-line" %> <%= t("actions.resend_invitation", scope: "decidim.admin") %>
+                      <% end %>
+                    </li>
+                  <% end %>
 
-              <% if allowed_to? :destroy, :assembly_user_role, user_role: role %>
-                <%= icon_link_to "delete-bin-line", assembly_user_role_path(current_assembly, role), t("actions.destroy", scope: "decidim.admin"), class: "action-icon--remove", method: :delete, data: { confirm: t("actions.confirm_destroy", scope: "decidim.admin") } %>
-              <% end %>
+                  <hr>
+
+                  <% if allowed_to? :update, :assembly_user_role, user_role: role %>
+                    <li class="dropdown__item">
+                      <%= link_to edit_assembly_user_role_path(current_assembly, role), class: "dropdown__button" do %>
+                        <%= icon "pencil-line" %> <%= t("actions.edit", scope: "decidim.admin") %>
+                      <% end %>
+                    </li>
+                  <% end %>
+
+                  <hr>
```

Comment:
Having this outside de above IF statement, could lead to a page where the menu displays 1-2 HRs one after another ( if the admin user is alowed to invite, and destory, but is not allowed by some magic to update ).